src/Symfony/Component/Notifier/Bridge/Octopush/OctopushTransportFactory.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Octopush;
+
+use Symfony\Component\Notifier\Exception\IncompleteDsnException;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class OctopushTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * @return OctopushTransport
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('octopush' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'octopush', $this->getSupportedSchemes());
```

Comment:
The class `UnsupportedSchemeException` should be updated according to this new scheme